rest-api-model-ql/src/main/kotlin/org/modelix/sample/restapimodelql/ModelServerLightWrapper.kt

```diff
@@ -0,0 +1,145 @@
+package org.modelix.sample.restapimodelql
+
+import University.Schedule.*
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.websocket.*
+import jetbrains.mps.lang.core.N_BaseConcept
+import kotlinx.coroutines.delay
+import org.modelix.client.light.LightModelClient
+import org.modelix.client.light.WebsocketConnection
+import org.modelix.metamodel.typed
+import org.modelix.model.repositoryconcepts.N_Module
+import org.modelix.model.repositoryconcepts.N_Repository
+import org.modelix.model.repositoryconcepts.models
+import org.modelix.model.repositoryconcepts.rootNodes
+import org.modelix.model.server.api.buildModelQuery
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+val logger: Logger = LoggerFactory.getLogger("org.modelix.sample.restapimodelql.ModelServerLightWrapper")
+
+class ModelServerLightWrapper {
```

Comment:
Parts of this class should probably be moved to the client?

Reply:
I think as this class provides all abstractions from the `LightModelClient` it fits as far as possible. not sure what would have to move?

Reply:
but we should take some of the solutions of the wrapper and add them to the client in the future

Reply:
for reference: https://github.com/orgs/modelix/projects/1/views/11?pane=issue&itemId=19623089